ocw/dataset_processor.py

```diff
@@ -391,8 +391,11 @@ def subset(target_dataset, subregion, subregion_name=None, extract=True, user_ma
     '''
 
     if not subregion.start:
-        subregion.start = target_dataset.times[0]
-        subregion.end = target_dataset.times[-1]
+        time_start = target_dataset.times[0]
```

Comment:
Thanks, @agoodm . I defined time_start and time_end not to change the subregion object in case the object is used again for subsetting another OCW dataset. I will update the test.